dry-operation.gemspec

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+Gem::Specification.new do |spec|
+  spec.name = "dry-operation"
+  spec.version = "0.0.0"
+  spec.authors = ["Tim Riley", "Marc Busque", "Brooke Kuhlmann"]
+  spec.email = ["tim@riley.id.au", "marc@lamarciana.com", "brooke@alchemists.io"]
+  spec.homepage = "https://dry-rb.org/gems/dry-operation"
+  spec.summary = "A domain specific language for composable business transaction workflows."
+  spec.license = "MIT"
+
+  spec.metadata = {
+    "bug_tracker_uri" => "https://github.com/dry-rb/dry-operation/issues",
+    "changelog_uri" => "https://github.com/dry-rb/dry-operation/blob/main/CHANGELOG.md",
+    "documentation_uri" => "https://dry-rb.org/gems/dry-operation",
+    "funding_uri" => "https://github.com/sponsors/hanami",
+    "label" => "Dry Operation",
```

Comment:
It's standard of [Gemsmith](https://alchemists.io/projects/gemsmith) and also utilized by the [Spek](https://alchemists.io/projects/spek) gem for presentation purposes. Essentially, used by all of my [projects](https://alchemists.io/projects) for static web page generation. It serves as a kind of _inflector_ since a gem name won't always translate from snake case to titleized text. It's also something I wish was part of the RubyGems specification because so many gem names don't match their associated labels and this gives us all a way to do that cleanly. :tada:

The reason you don't see this in the RubyGems specification is because the specification allows for freeform key/value pairs so I'm making use of what the specification allows. It's a great feature.

So essentially, we now get this:

- **Gem Name**: dry-operation
- **Gem Label**: Dry Operation

Much like how I render all Dry gems in the [Pennyworth UI](https://alchemists.io/projects/pennyworth#_dry_rb):

![dry](https://github.com/dry-rb/dry-operation/assets/50245/c4e288fa-3e8e-48a0-b34b-b2720c625cf5)

Reply:
Ah, thanks for the explainer! I'm more than happy to see this stay in, but could you please rename it to "dry-operation"? Those lower-cased dasherized forms are the official names of our gems.

Reply:
Amended.

I didn't realize, until now, that the name and label are meant to be the same. Definitely bums me out since I like the presentation of software to be human readable/friendly without having the code/name bleed through. Whenever I see projects titled like that, I always read it as a typo. 😅